test/e2e/azure_privatecluster.go

```diff
@@ -107,23 +107,21 @@ func AzurePrivateClusterSpec(ctx context.Context, inputGetter func() AzurePrivat
 	}, "5s", "100ms").Should(BeNil(), "Failed to assert public API server stability")
 
 	// **************
-	spClientSecret := os.Getenv("AZURE_CLIENT_SECRET")
-	secret := &corev1.Secret{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "cluster-identity-secret-private",
-			Namespace: input.Namespace.Name,
-		},
-		Type: corev1.SecretTypeOpaque,
-		Data: map[string][]byte{"clientSecret": []byte(spClientSecret)},
+	// Get the Client ID for the user assigned identity
+	subscriptionID := os.Getenv(AzureSubscriptionId)
+	identityRG := os.Getenv(AzureIdentityResourceGroup)
```

Comment:
We could probably do this instead:
```suggestion
	identityRG, ok := os.Getenv(AzureIdentityResourceGroup)
	if !ok {
		identityRG = "capz-ci"
	}
```

Reply:
switched to LookupEnv